pandas/core/indexes/base.py

```diff
@@ -1471,6 +1471,38 @@ def is_object(self):
         return is_object_dtype(self.dtype)
 
     def is_categorical(self):
+        """
+        Check if the Index type is categorical.
+
+        The given object must be a pandas DataFrame index or a pandas Series
```

Comment:
I think it's best to just remove this. There isn't really an object being passed, it's just an attribute on Index classes.